cms/djangoapps/contentstore/management/commands/migrate_cert_config.py

```diff
@@ -0,0 +1,257 @@
+            active_version_collection.aggregate(
+                [
+                    {
+                        '$group': {
+                            '_id': 1,
+                            'draft': {'$push': '$versions.draft-branch'},
+                            'published': {'$push': '$versions.published-branch'},
+                        }
+                    },
+                    {'$project': {'_id': 1, 'branches': {'$setUnion': ['$draft', '$published']}}},
+                ]
+            )
+        )[0]['branches']
+
+        structures = structure_collection.find(
+            {
+                '_id': {'$in': branches},
+                'blocks': {
+                    '$elemMatch': {
+                        '$and': [
+                            {'block_type': 'course'},
+                            {'fields.certificates': {'$exists': True}},
+                        ]
+                    }
+                },
+            },
+            {
+                '_id': True,
+            },
+        )
+
+        structure_ids = [struct['_id'] for struct in structures]
+        split_mongo_courses = list(
+            active_version_collection.find(
+                {
+                    '$or': [
+                        {'versions.draft-branch': {'$in': structure_ids}},
+                        {'versions.published': {'$in': structure_ids}},
+                    ]
+                },
+                {
+                    'org': True,
+                    'course': True,
+                    'run': True,
+                    'versions': True,
+                },
+            )
+        )
+        for course in split_mongo_courses:
+            yield split_modulestore.make_course_key(
+                course['org'],
+                course['course'],
+                course['run'],
+            )
+
+    def send_to_credentials(
+        self,
+        course_key: CourseKey,
+        mode: CourseMode,
+        certificate_data: Dict[str, Union[str, List[str]]]
+    ):
+        """
+        Sends certificate configuration data to Credential IDA via http request.
+        """
+        certificate_config = create_course_certificate_config_data(str(course_key), mode.slug, certificate_data)
+        files_to_upload = dict(get_certificate_signature_assets(certificate_config))
+        certificate_config_data = attr.asdict(certificate_config)
+        send_course_certificate_configuration(str(course_key), certificate_config_data, files_to_upload)
+
+    def delete_from_store(self, course: CourseBlock, certificates: List[Dict[str, str]]):
+        """
+        Deletes certificate configuration from modulestore storage.
+        """
+        store = modulestore()
+        request = FakedRequest(ModuleStoreEnum.UserID.mgmt_command)
+        for cert in certificates:
+            CertificateManager.remove_certificate(
+                request=request, store=store, course=course, certificate_id=cert['id']
+            )
```

Comment:
It's okay to leave this functionality in, since you've already written it. I just want to be cautious in using it.

Reply:
Yes, here we are talking about the fact that you need to remove what is no longer needed. I thought so and it was the main idea.

We have successfully (here the key is successful) transferred all the information from Mongo, and this can be quite a lot of disk space and we are clearing this disk space so that it is not dead ballast that everyone is afraid to touch and do not know what to do with it. 
his option is enabled only by flag, so if a person makes his own decisions and takes full responsibility for himself. This may be useful to open source consumers of OpenEdx. For edx.org, this is not critical because of the importance of the information, the large volume and low cost of disk space, compared to the problems that can be gotten.

So If you want to delete, then delete deliberately. If something goes wrong, then database dumps can help here I guess.